src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -8018,7 +8018,18 @@ def test_performance_plus(self, resource_group):
         self.cmd('disk create -n {disk} -g {rg} --size-gb 530 --performance-plus', checks=[
             self.check('creationData.performancePlus', True)
         ])
+    @ResourceGroupPreparer(name_prefix='cli_test_performance_plus_update_', location='eastus2euap')
```

Comment:
```suggestion

    @ResourceGroupPreparer(name_prefix='cli_test_performance_plus_update_', location='eastus2euap')
```

Reply:
Actually, `test_performance_plus` and `test_performance_plus_update` are testing the same feature, could you combine them into one test?